Components/kube-ui.css

```diff
@@ -0,0 +1,40 @@
+@import './avatar.css';
+@import './badge.css';
+
```

Comment:
You can add comment like /** external imports**/ for sanity

Reply:
Done